t/cmd/git-credential-lfstest.go

```diff
@@ -89,7 +89,12 @@ func fill() {
 }
 
 func credsForHostAndPath(host, path string) (string, string, error) {
-	hostFilename := filepath.Join(credsDir, host)
+	var hostFilename string
+	if host != "" {
```

Comment:
I fought with this code quite extensively to figure out why it wasn't working as I expected it to.  

Come to think of it, I probably need a comment here, so the next person to come along doesn't try to “fix” it.  It will definitely break, but it won’t be obvious why.